pages/dashboard.html

```diff
@@ -0,0 +1,139 @@
+                    <div class="card-content">
+                        <h3 class="card-title">aaa</h3>
+                        <p class="card-description">info</p>
+                    </div>
+                </div>
+                <div class="card">
+                    <div class="card-content">
+                        <h3 class="card-title">aaa</h3>
+                        <p class="card-description">info</p>
+                    </div>
+                </div>
+                <div class="card">
+                    <div class="card-content">
+                        <h3 class="card-title">aaa</h3>
+                        <p class="card-description">info</p>
+                    </div>
+                </div>
+                <div class="card">
+                    <div class="card-content">
+                        <h3 class="card-title">aaa</h3>
+                        <p class="card-description">info</p>
+                    </div>
+                </div>
+                <div class="card">
+                    <div class="card-content">
+                        <h3 class="card-title">aaa</h3>
+                        <p class="card-description">info</p>
+                    </div>
+                </div>
+                <div class="card">
+                    <div class="card-content">
+                        <h3 class="card-title">aaa</h3>
+                        <p class="card-description">info</p>
+                    </div>
+                </div>
+                <div class="card">
+                    <div class="card-content">
+                        <h3 class="card-title">aaa</h3>
+                        <p class="card-description">info</p>
+                    </div>
+                </div>
+                <div class="card">
+                    <div class="card-content">
+                        <h3 class="card-title">aaa</h3>
+                        <p class="card-description">info</p>
+                    </div>
+                </div>
+                <div class="card">
+                    <div class="card-content">
+                        <h3 class="card-title">aaa</h3>
+                        <p class="card-description">info</p>
+                    </div>
+                </div>
+                <div class="card">
+                    <div class="card-content">
+                        <h3 class="card-title">aaa</h3>
+                        <p class="card-description">info</p>
+                    </div>
+                </div>
+                <div class="card">
+                    <div class="card-content">
+                        <h3 class="card-title">aaa</h3>
+                        <p class="card-description">info</p>
+                    </div>
+                </div>
+                <div class="card">
+                    <div class="card-content">
+                        <h3 class="card-title">aaa</h3>
+                        <p class="card-description">info</p>
```

Comment:
in dashbord, in signup page and in forgot your password page:
please add the cursor property to specifie the mouse cursor to be displayed when pointing over the element